### DISCUSSION TITLE
How to use Authz endpoints?

### DISCUSSION THREAD

**Discussion by sarpuser** [authelia/authelia#7777] [Q&A] [answered]
I have some services that are protected behind Authelia. These are applications have phone apps that can connect to them (from outside the network) with the URL and an API key. The issue I am running into is that because these services are behind Authelia, the apps can't actually connect to the services.

Is there a way to get some sort of  Auth token that I could append to the URLs to authorize access, without my Authelia password and 2FA? Is that what the Authz endpoints are? The documentation has configuration examples but I don't really understand how to use them. **Also I do not want to bypass Authelia for these services.** Any help would be much appreciated.

**Comment by christaikobo** [comment, marked as answer]
Off the top of my head I think there might be 2 ways to do it.

1. I presume you have a reverse https proxy in front of your services. Let's say you have a http server running at 10.0.0.5:8080, and you reverse proxy it at https://example.com/mywebserver, you can add a secondary reverse proxy entry to serve it at https://example.com/mywebserverformyappssuperlongpassword. Then you add a policy in authelia to bypass resource ^/mywebserverformyappssuperlongpassword$. Since URL is encrypted in a https request, your services should be safe. 

2. You can use basic auth for those services, I don't know how your apps handle basic auth, maybe it'll work.
